cfme/tests/infrastructure/test_infra_manual.py

```diff
@@ -82,3 +82,249 @@ def test_provider_summary_topology():
         1532404
     """
     pass
+
+
+@pytest.mark.manual
+@test_requirements.infra_hosts
+def test_setting_default_filter():
+    """
+    Verify the creation and proper functionality of default filters.
+
+    Polarion:
+        assignee: prichard
+        casecomponent: Infra
+        caseimportance: low
+        initialEstimate: 1/8h
+        testSteps:
+            1. Navigate to the Compute > Infrastructure > Hosts view.
+            2. Select a filter that does not have "(Default)" appended to the name. Note items
```

Comment:
I would say `select a filter from the accordion`, I got confused between this and the search bar.